.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,6 +1,7 @@
 blank_issues_enabled: false
 contact_links:
-  - name: Join Our Slack Channel wg-manifests.
+  - name: Join Our CNCF Slack Channel kubeflow-platform..
```

Comment:
[#kubeflow-platfrom](https://app.slack.com/client/T08PSQ7BQ/C073W572LA2) links directly to the channel